frontend/packages/gitops-plugin/src/components/details/GitOpsDetails.tsx

```diff
@@ -90,6 +94,17 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs, appName }) => {
           </Alert>
         </>
       )}
+      {error != null && (
+        <>
```

Comment:
Is this Fragment is required here? Look like it should work without it.

Reply:
Good catch. I removed the `<></>`  from the above section as well